platform/ios/src/MGLMapView.mm

```diff
@@ -2305,14 +2326,9 @@ - (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
             CGPoint west = [panGesture locationOfTouch:0 inView:panGesture.view];
             CGPoint east = [panGesture locationOfTouch:1 inView:panGesture.view];
             
-            if (west.x > east.x) {
-                CGPoint swap = west;
-                west = east;
-                east = swap;
-            }
-            
-            CLLocationDegrees horizontalToleranceDegrees = 60.0;
-            if ([self angleBetweenPoints:west east:east] > horizontalToleranceDegrees) {
+            CLLocationDegrees horizontalToleranceDegrees = 45.0;
```

Comment:
Should this be made a constant, given that it’s also used above in the `handleTwoFingerDragGesture:` angle check?